xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/editors/dataeditors.js

```diff
@@ -133,6 +133,9 @@ editors.XDataEditors = Class.create({
               },
               onComplete : function() {
                 item.disabled = false;
+                require(['xwiki-meta'], function (xm) {
```

Comment:
Could we wrap the whole ajax request into the require() call, so we are sure that everything is synchronized?

Reply:
I wasn't quite sure but I can give it a try.